fe/fe-core/src/main/java/com/starrocks/clone/DiskAndTabletLoadReBalancer.java

```diff
@@ -316,7 +316,7 @@ private void checkAndUseWorkingSlots(long beId, long pathHash,
         if (srcBePathSlot == null) {
             throw new SchedException(SchedException.Status.UNRECOVERABLE, "working slots not exist for src be");
         }
-        if (srcBePathSlot.takeBalanceSlot(pathHash) == -1) {
+        if (srcBePathSlot.takeSlot(pathHash) == -1) {
```

Comment:
How to test this strategy works?

Reply:
The current balance/repair. Do we have case to test it?

Reply:
I have added test cases in PseudoCluster.